common/config/webpack/sampleapp.webpack.config.js

```diff
@@ -51,9 +51,11 @@ const webpackConfig = (sampleAppDir, env) => ({
     new webpack.DefinePlugin({
       'process.env.PRODUCTION': env.production || !env.development,
       'process.env.NAME': JSON.stringify(require(path.resolve(sampleAppDir, 'package.json')).name),
-      'process.env.VERSION': JSON.stringify(require(path.resolve(sampleAppDir, 'package.json')).version)
+      'process.env.VERSION': JSON.stringify(require(path.resolve(sampleAppDir, 'package.json')).version),
+      __CALLINGVERSION__: JSON.stringify(require(path.resolve(sampleAppDir, 'package.json')).dependencies['@azure/communication-calling']),
+      __CHATVERSION__: JSON.stringify(require(path.resolve(sampleAppDir, 'package.json')).dependencies['@azure/communication-chat']),
```

Comment:
just want to confirm this is a webpack config shared by all of the samples and thats why we have a mention of the chat version even tho we are only changing the calling sample in this change?

Reply:
Yep its shared, so for calling this will just resolve to undefined (and __CHATVERSION__ isn't used anywhere in calling).
(You're right though its not necessary for this PR - just typed them both quickly)